packages/streamline/src/components/list-items/list-item-option/list-item-option.type.ts

```diff
@@ -0,0 +1,25 @@
+import { IconsName } from '../../../primitives/icon/icon.types';
```

Comment:
The file should be name `.types.ts` to be consistent with the other. If you have spotted discrepancies, we can take this opportunity to fix and align.